### PR TITLE
Building an article based on New York Times.

### PR DESCRIPTION
**Pull request by kcotrinam**
_**For this project we:**_

-Added the HTML structure based on W3Cschool semantic.
-Added CSS variable for colors and font sizes.
-Added general classes used over the different HTML elements.
-Validate the HTML and CSS with lighthouse linter and style linter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# NewYorkTimes-article
\ No newline at end of file
+# New York Times article
+
+This project is based on the real Youtube website. It was design in order to practice HTML and css. It has a basic structure, such us a navbar, a section, an aside and a footer.
```
Comment (MuriloRoque) [inline README.md]: YouTube? 😅

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+  display: inline-block;
+}
+
+img {
+  width: 100%;
+}
+
+.width-100 {
+  width: 100%;
+}
```
Comment (MuriloRoque) [inline css/styles.css:35]: You can select these at the same time 😉

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested 🔴 

## Hello @kcotrinam92 and @len23 👋 

Your page is very nice, I really like it, congratulations for your hard work in this project 👏 👏 👏 

However, we still need to change somethings, please follow my suggestions:



# Page

## Navigation bar

- [x] Your NYT logo should be a link, like in the original:

logo-link
<br>
- [x] You need to work a little bit more in the right part, with the 'Subscribe' button. Make it look like the original and make them links, please:

**Yours:**
yours-login
<br>
**Original:**
original-login
<br>
### MainContent
- [x] I can't see your 'Editor's Picks' section, please implement it (please use < aside > tag 😄 ):

editor's picks

<br>

- [x] At the bottom of the text, we can see the big 'comments' section and some icons. These should also be links, like the original:

yours-bottom

### Bottom part

- [ ] Your images and texts here should all be links, except for the date, like the original:
yours-bottom-imgs

- [x] Please, check these for links too, see what is a link and what isn't in the original:

yours-bottom-foot

# General

- [x] Please check your linter errors check, by the time I was reviewing this project, it wasn't finished.

- [x] Please check my comments in your code below.

# Summary

- Your code is awesome, your HTML is semantic and you are using CSS best practices 👍 
- Your README file, Pull Request and commit messages are very professional 💯 
- Your page is very well structured, we just need to fix some details 🔝 

thumbs



Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

## I really like your project! Congrats @kcotrinam92 and @len23 , you guys did an excellent job doing the previously requested changes asked by our great TSE @MuriloRoque !

### _**Clean and modular code, following the best practices and the design a looks great, keep it up!**_ 🥇

_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio ✨
  
### [OPTIONAL CHANGES]
- [ ] Kindly remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) 
For example: Create one only class like this: 
```
.black { 
   color: #000 
}
``` 
and use that same class in your HTML files when you need it.

- Frequency of repeated rules:
image

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**